### PR TITLE
Enabling encryption topic, content from Alex

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/tasks/secure/encryption/_index.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  * Amazon Elastic Kubernetes Service (EKS)
+  * DigitalOcean Kubernetes (DOKS)
+  * Google Kubernetes Engine (GKE) in a Google Cloud project
+  * Microsoft Azure Kubernetes Service (AKS)
+* K8ssandra version 2.0 or later
+* An SSL encryption store, as covered in the next section
+
+## Generating SSL encryption stores
+
+If you do not have a set of encryption stores available, follow the instructions in [this TLP blog post](https://thelastpickle.com/blog/2021/06/15/cassandra-certificate-management-part_1-how-to-rotate-keys.html). More specifically, use [the script](https://github.com/thelastpickle/cassandra-toolbox/tree/main/generate_cluster_ssl_stores) used to generate the SSL stores.
```
Comment (adejanovski) [inline docs/content/en/tasks/secure/encryption/_index.md:23]:
```suggestion
If you do not have a set of encryption stores available, follow the instructions in [this TLP blog post](https://thelastpickle.com/blog/2021/06/15/cassandra-certificate-management-part_1-how-to-rotate-keys.html). More specifically, use [this script](https://github.com/thelastpickle/cassandra-toolbox/tree/main/generate_cluster_ssl_stores) to generate the SSL stores.
```

File: docs/content/en/tasks/secure/encryption/_index.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+In order to create a K8ssandra cluster with encryption, first create a namespace and the encryption stores secrets previously generated in it.
+
+In the `K8ssandraCluster` manifest, you will need to configure encryption settings in the `config/cassandraYaml` section.
+
+Also, you'll need to reference the encryption stores' secrets under:
+
+* `cassandra/serverEncryptionStores` 
+* *Or* `cassandra/clientEncryptionStores`
+
+We emphasized "or" here because, theoretically, the keystore and truststore could be placed in different secrets.
```
Comment (adejanovski) [inline docs/content/en/tasks/secure/encryption/_index.md:81]: server encryption and client encryption are different things and they both have their own keystore/truststore pair.
The "or" here shows that you can turn on either independently, or both.
Server is for internode communications encryption, and client is for client to node communications encryption.

It is true that truststore and keystore can be in different secrets, but its unrelated to the type of encryption we're dealing with.

**Review by adejanovski — CHANGES_REQUESTED** [review]
There's a little confusion around server/client encryption and the encryption stores secrets it seems.

File: docs/content/en/tasks/secure/encryption/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/tasks/secure/encryption/_index.md]: Inc'd comments - thanks Alex.

**Pull request merged**